modules/outbrainBidAdapter.js

```diff
@@ -62,7 +66,8 @@ export const spec = {
     const publisher = setOnAny(validBidRequests, 'params.publisher');
     const bcat = setOnAny(validBidRequests, 'params.bcat');
     const badv = setOnAny(validBidRequests, 'params.badv');
-    const eids = setOnAny(validBidRequests, 'userIdAsEids')
+    const eids = setOnAny(validBidRequests, 'userIdAsEids');
+    const wlang = setOnAny(validBidRequests, 'params.wlang');
```

Comment:
Wlang bcat and badv are all ortb2 params, please also check the ortb2 part of the bid request object.

Reply:
Thank you for the review

We're now using wlang from ortb2 object, and we'll default to bcat and badv from ortb2 as well.
I don't see this documented at https://docs.prebid.org/features/firstPartyData.html though

How does it look now?

Reply:
Looks good

https://github.com/prebid/Prebid.js/issues/7925